example-config.json

```diff
@@ -1,135 +1,207 @@
+            "Admin": "",
+            "Moderator": "",
+            "Helper": "",
+            "Staff": "",
+            "NewTicket": "",
+            "BetaTester": "",
+            "Booster": "",
+            "Donator": "",
+            "Client": "",
+            "Member": "",
+            "Bot": "",
+            "Muted": "",
+            "TicketBan": "",
+            "Announcements": "",
+            "Changelogs": "",
+            "Giveaways": "",
+            "Polls": "",
+            "GameNights": "",
+            "MovieNights": "",
+            "StaffUpdates": "",
+            "Others": ""
         }
-      }
-    }
-  },
-  "DB": {
-    "MongoDB": "mongodb - unused"
-  },
-  "Pterodactyl": {
-    "hosturl": "https://panel.danbot.host",
-    "apikey": "",
-    "apikeyclient": ""
-  },
-
-     "Email": {
-    "Host": "mail.danbot.host",
-    "Port": "587",
-    "User": "no-reply@danbot.host",
-    "Password": "",
-    "From": "no-reply@danbot.host"
-  },
-    
-  "Email2": {
-    "Host": "smtp.sendgrid.net",
-    "Port": "587",
-    "User": "apikey",
-    "Password": "",
-    "From": "noreply@danbot.host"
-  },
-
-  "Nodes": {
-      "PNode1": "",
-      "PNode2": "",
-      "PNode3": "",
-      "PNode4": "",
-      "Dono1": "",
-      "Dono2": "",
-      "Dono3": ""
-  },
-  "Servers": {
-      "KS-US-1": "",
-      "DE-EU-1": "",
-      "DE-EU-2": "",
-      "DE-EU-3": "",
-      "LO-UK-1": ""
-  },
-  "Tunnels": {
-      "US-1": "",
-      "UK-1": ""
-  },
-  "VPN": {
-      "US-1": "",
-      "UK-1": "",
-      "EU-1": "",
-      "AU-1": "",
-      "FR-1": ""
-  }
-}
+    },
+    "DB": {
+        "MongoDB": ""
+    },
+    "Pterodactyl": {
+        "hosturl": "",
+        "apikey": "",
+        "apikeyclient": ""
+    },
+    "PrivPterodactyl": {
+        "hosturl": "",
+        "apikey": "",
+        "apikeyclient": ""
+    },
+    "DanPterodactyl": {
+        "hosturl": "",
+        "apikeyclient": ""
+    },
+    "Email": {
+        "Host": "",
+        "Port": "",
+        "User": "",
+        "Password": "",
+        "From": ""
+    },
+    "Email2": {
+        "Host": "",
+        "Port": "",
+        "User": "",
+        "Password": "",
+        "From": ""
+    },
+    "Nodes": {
+        "PNode1": "",
+        "PNode2": "",
+        "PNode3": "",
+        "PNode4": "",
+        "Dono1": "",
+        "Dono2": "",
+        "Dono3": "",
+        "Dono4": "",
+        "Storage1": ""
+    },
+    "Servers": {
+        "US1": "",
+        "EU1": ""
+    },
+    "Tunnels": {
+        "US-1": "",
+        "UK-1": ""
+    },
+    "VPN": {
+        "US1": "",
+        "UK-1": "",
+        "EU-1": "",
+        "AU1": "",
+        "FR1": ""
+    },
+    "Services": {
+        "grafana": "",
+        "pterocore": "",
+        "pteropublic": "",
+        "vpnapi": ""
+    },
+    "BannedNames": [
+        "",
+        "",
+        "",
+        "",
+        ""
+    ],
+    "Enabled": {
+        "NodeStats": "",
+        "Welcome": "",
```

Comment:
Handled by beta bot

Reply:
No, NodeStats is still used in the code.